crates/polars/src/docs/lazy.rs

```diff
@@ -112,10 +112,10 @@
 //! use polars::prelude::*;
 //! # fn example() -> PolarsResult<()> {
 //!
-//!  let df = LazyCsvReader::new("reddit.csv")
+//!  let df = LazyCsvReader::new()
```

Comment:
This diff shows how the public API differs. By moving the path/paths/glob/etc to be loaded to the `load_*` methods, there is no longer a requirement for weird internal state tracking of this info.